### PR TITLE
Update vpnaas docs

### PR DESCRIPTION
**Pull request by morozovalekseywot**
Update vpnaas docs

### CODE REVIEW

File: examples/vpnaas/service/main.tf [hunk 1]
```diff
@@ -1,4 +1,7 @@
 resource "vkcs_vpnaas_service" "vpn_to_datacenter" {
   name      = "vpn-tf-example"
+
+  # See the argument description and check vkcs_networks_sdn datasource output to figure out
+  # what type of router you should use in certain case (router or dc_router)
```
Comment (ftersin) [inline examples/vpnaas/service/main.tf:5]: lets specify full resource type names, w/ vkcs_networks prefix

File: vkcs/vpnaas/resource_endpoint_group.go [hunk 2]
```diff
@@ -58,7 +58,7 @@ func ResourceEndpointGroup() *schema.Resource {
 				Computed:    true,
 				Optional:    true,
 				ForceNew:    true,
-				Description: "The type of the endpoints in the group. A valid value is subnet, cidr, network, router, or vlan. Changing this creates a new group.",
+				Description: "The type of the endpoints in the group. A valid value is subnet and cidr. For sprut SDN only cidr can be used, for Neutron SDN - subnet. Changing this creates a new group.",
```
Comment (ftersin) [inline vkcs/vpnaas/resource_endpoint_group.go:61]: for neutron - cidr for remote group, subnet for local

File: examples/vpnaas/service/main.tf (hunk 1 above)
Comment (morozovalekseywot) [reply examples/vpnaas/service/main.tf]: done

File: vkcs/vpnaas/resource_endpoint_group.go (hunk 2 above)
Comment (morozovalekseywot) [reply vkcs/vpnaas/resource_endpoint_group.go]: done

File: vkcs/vpnaas/resource_endpoint_group.go [hunk 3]
```diff
@@ -58,7 +58,7 @@ func ResourceEndpointGroup() *schema.Resource {
 				Computed:    true,
 				Optional:    true,
 				ForceNew:    true,
-				Description: "The type of the endpoints in the group. A valid value is subnet, cidr, network, router, or vlan. Changing this creates a new group.",
+				Description: "The type of the endpoints in the group. A valid value is subnet and cidr. For sprut SDN only cidr can be used, for neutron SDN - cidr for remote group, subnet for local. Changing this creates a new group.",
```
Comment (ftersin) [inline vkcs/vpnaas/resource_endpoint_group.go:61]: lets surround choices w/ ` commas

File: vkcs/vpnaas/resource_service.go [hunk 4]
```diff
@@ -70,7 +70,7 @@ func ResourceService() *schema.Resource {
 				Required:    true,
 				Computed:    false,
 				ForceNew:    true,
-				Description: "The ID of the router. Changing this creates a new service.",
+				Description: "The ID of the router. Use router id for Neutron SDN and dc_router id for sprut SDN. To get a list of available SDNs in a project, you can use vkcs_networking_sdn datasource. Changing this creates a new service",
```
Comment (ftersin) [inline vkcs/vpnaas/resource_service.go:73]: lets surround datasource name w/ ` comma

File: vkcs/vpnaas/resource_endpoint_group.go (hunk 3 above)
Comment (morozovalekseywot) [reply vkcs/vpnaas/resource_endpoint_group.go]: done

**Review by ftersin — APPROVED** [review]

**Pull request merged**